app/src/main/java/org/simlar/helper/CallConnectionDetails.java

```diff
@@ -129,12 +129,12 @@ public int getQualityDescription()
 		return mQuality.getDescription();
 	}
 
-	public String getCodec()
+	public CharSequence getCodec()
```

Comment:
CharSequence "does not refine the general contracts of the equals and hashCode methods. The result of comparing two objects that implement CharSequence is therefore, in general, undefined." We want comparable strings, in general, even if we do not actually compare them, yet.